cmd/cloud/server.go

```diff
@@ -85,11 +87,21 @@ var serverCmd = &cobra.Command{
 			"working-directory": wd,
 		}).Info("Starting Mattermost Provisioning Server")
 
+		privateSubnetIds, _ := command.Flags().GetString("private-subnets")
+		logger.Infof("Using private subnets %s", privateSubnetIds)
+		publicSubnetIds, _ := command.Flags().GetString("public-subnets")
+		logger.Infof("Using public subnets %s", publicSubnetIds)
+
+		route53ZoneID, _ := command.Flags().GetString("route53-id")
+
 		// Setup the provisioner for actually effecting changes to clusters.
 		kopsProvisioner := provisioner.NewKopsProvisioner(
 			clusterRootDir,
 			s3StateStore,
+			aws.New(route53ZoneID),
```

Comment:
@jwilander In my PR https://github.com/mattermost/mattermost-cloud/pull/29 I am already passing an AWS client for private hosted zone ID, which we need for DNS registration. I guess you can use the same client for AWS related stuff.